### ISSUE TITLE
Keyboard shortcut/right click for encryption

### ISSUE THREAD

**Issue by ghost** [mailvelope/mailvelope#205]
> some drive-by promotion of mailvelope prompts this issue:
> 
> ```
> why can't we open the encryption window via keyboard shortcut or right click?
> ```

can we build in keyboard shortcuts? seems far fetched, but awesome.

**Comment by ironside** [comment]
When I'm typing an email I'm using the keyboard. When I'm done with the email I press CTR+ENTER to send it. I would like to be able to navigate encrypting an email just as easily with the keyboard. Without keyboard shortcuts the extra effort is enough to keep me from using this otherwise wonderful extension.

**Comment by ironside** [comment]
Indeed, even here in github you submit a comment with CTRL+ENTER (no mouse clicking required)
